src/test/java/e2e/pages/EditRecordCourse.java

```diff
@@ -0,0 +1,72 @@
+package e2e.pages;
+
+import e2e.wait.Wait;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class EditRecordCourse extends PageBase{
```

Comment:
You can do extends AddCoursePage,  EditRecordCourse had methods same like AddCoursePage , 
on this page can stay methods 

```
public void waitForLoading(){
        wait = new Wait(driver);
        wait.forVisibility(editDialog);
    }

public void clickCancelButton(){
        click(cancelButton);
    }
    public void clickSaveButton(){
        click(saveButton);
    }
```
Other method you can remove